src/components/ProgramRecordAlert/ProgramRecordAlert.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Alert, Button, Hyperlink } from '@edx/paragon';
+import { Info, CheckCircle } from '@edx/paragon/icons';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { logError } from '@edx/frontend-platform/logging';
+
+import sendRecords from '../ProgramRecordSendModal/data/service';
+
+const ProgramRecordAlert = ({
+  alertType, onClose, programId, username, setSendRecord, creditPathway,
+}) => {
+  const handleTryAgainSendRecord = async () => {
+    await sendRecords(programId, username, creditPathway.id)
+      .then(response => {
+        if (response.status >= 200 && response.status < 300) {
+          setSendRecord(prev => ({
+            ...prev,
+            sendRecordSuccessOrgs: [...prev.sendRecordSuccessOrgs, creditPathway],
+            sendRecordFailureOrgs: prev.sendRecordFailureOrgs.filter(pathway => pathway.id !== creditPathway.id),
+          }));
+        }
+      })
+      .catch(error => {
+        const errorMessage = (`Error: Could not send record again: ${error.message}`);
+        logError(errorMessage);
+      });
+  };
+
+  const getAlert = () => {
+    switch (alertType) {
+      case 'failure':
+        return (
+          <Alert
+            variant="danger"
+            icon={Info}
+            actions={[
+              <Button
+                onClick={handleTryAgainSendRecord}
+              >
+                <FormattedMessage
+                  id="send.records.try.again.button"
+                  defaultMessage="Try Again"
+                  description="Button that attempts to send the records again"
+                />
+              </Button>,
+            ]}
+            dismissible
+            onClose={() => onClose(creditPathway.id)}
+          >
+            <Alert.Heading>
+              <FormattedMessage
+                id="send.failure.alert.heading"
+                defaultMessage="We were unable to send your program record"
+                description="Heading for the alert that displays when the program record fails to send"
+              />
+            </Alert.Heading>
+            <FormattedMessage
+              id="send.failure.alert.message"
+              defaultMessage="You can try to send your record to the {pathway_name} pathway again. If this issue persists"
+              description="Message for the alert that displays when the program record fails to send"
+              values={{
+                pathway_name: creditPathway.name,
+              }}
+            />
+            <Hyperlink
+              destination="https://support.edx.org/hc/en-us/sections/360001216693-Learner-records"
+              target="_blank"
+              showLaunchIcon={false}
+            >
+              <FormattedMessage
+                id="contact.support.link"
+                defaultMessage="contact edX Support."
+                description="Link that directs the user to edX support page"
```

Comment:
We should update the description to something like: "Link that directs the user to their organization/platform/something support page".